packages/core/src/components/popover-core/tds-popover-core.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../global/motion';
+
+:host {
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity $popover-core-transition;
```

Comment:
isn't this getting added no matter if animation is set to fade or none?

Reply:
Probably. We might need to have two classes to control this. One to add animation rule like `.tds-popover-animation` and another one like `.tds-popover-animation--action` that will trigger the changes and animation.

Reply:
Good input! I update the animation class according to your suggestions. 
To maintain modularity and clarity in the code this is great suggestions.